freddytoolapi/Freddy.UnitTests/Utilities/HttpClientExtensions.cs

```diff
@@ -12,13 +12,21 @@ public static class HttpClientExtensions
         {
             PropertyNamingPolicy = JsonNamingPolicy.CamelCase
         };
-        
-        public static async Task<TObject> GetObjectAsync<TObject>(this HttpClient client, string url)
+
+        public static async Task<TObject> GetObjectAsync<TObject>(this HttpClient client, string url) where TObject : class
         {
             var response = await client.GetStreamAsync(url);
-            return await JsonSerializer.DeserializeAsync<TObject>(response, Options);
+
+            try
+            {
+                return await JsonSerializer.DeserializeAsync<TObject>(response, Options);
+            }
+            catch
+            {
+                return null;
```

Comment:
skus default

Reply:
povodne to bolo default ale potom som skusil ze ak dam constraint na class (reference value) tak tam moze ist null a ano mas pravdu int neurobi , ale teoreticky v tomto pripade nechcem , teda zatial robit s value type takze to bola taka own decision, ale vratim to na value s defaul aby som se nedobomboval v buducnosti kvoli tomu.